scripts/manually-run-nightly-tests.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+set -eu
+
+TOKEN=$(find "$HOME" -type f -maxdepth 2 -exec grep -l "^ghp_" {} \; -quit | xargs cat)
+TEST_ENV="perf"
+
+# ensure the script always runs in the parent directory terra-data-catalog
+CWD=$(dirname "$(dirname "$(readlink -f "$0")")")
+cd "$CWD"
+
+# get the helm chart versions for the perf env
+curl -H "Authorization: token ${TOKEN}" \
+    -H 'Accept: application/vnd.github.v3.raw' \
+    -L https://api.github.com/repos/broadinstitute/terra-helmfile/contents/versions/app/dev.yaml \
+    --create-dirs -o "integration/src/main/resources/rendered/dev.yaml"
+curl -H "Authorization: token ${TOKEN}" \
+    -H 'Accept: application/vnd.github.v3.raw' \
+    -L https://api.github.com/repos/broadinstitute/terra-helmfile/contents/environments/live/perf.yaml \
+    --create-dirs -o "integration/src/main/resources/rendered/$TEST_ENV.yaml"
+
+# render the service account secrets
+vault read secret/dsde/firecloud/dev/common/firecloud-sa -format=json | \
+    jq -r .data.key | base64 -d > "integration/src/main/resources/rendered/user-delegated-sa.json"
+
+vault read secret/dsde/terra/kernel/perf/common/testrunner/testrunner-sa -format=json | \
+    jq -r .data.key | base64 -d > "integration/src/main/resources/rendered/testrunner-perf.json"
+
+# run the perf test suite
+./gradlew --build-cache runTest --args="suites/$TEST_ENV/FullIntegration.json build/reports"
+./gradlew --build-cache uploadResults --args="CompressDirectoryToTerraKernelK8S.json build/reports"
```

Comment:
Is `uploadResults` necessary for a manual test run? I'm not sure what the purpose of this script is; is it to test out the code in the GitHub workflow? Since it duplicates the code in the workflow, it may not be a great way to do that, as now we'll have to change things in both places to keep them in sync.

Reply:
As the GitHub Action has not been working since Friday afternoon, Gary recommended I run the steps manually. Instead of manually running each line in the GitHub Action I thought it would be more consistent running it in a shell script which duplicates the functionality. I agree with you generally we should avoid duplicating code, but this was more of a tool I created for myself to be able to run the steps without having to manually run each line. I thought I would share it with the team in case this happens again in the future, but I can remove it if you don't think it is useful. I think rewriting our action to use the shell script would require a lot more effort.

Reply:
Ok. I think it's OK to keep it in for now, although I'd rather remove it than maintain it if it's not something we need to use often.